ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -111,3 +132,57 @@ Wait For Pods Numbers
   IF    '${status}' == 'False'
         Run Keyword And Continue On Failure    FAIL    Timeout- ${output} pods found with the label selector ${label_selector} in ${namespace} namespace
   END
+
+Apply DataScienceCluster CustomResource
+    [Documentation]
+    [Arguments]        ${dsc_name}=default
+    ${file_path} =    Set Variable    ods_ci/tasks/Resources/Files/
+    Log to Console    Requested Configuration:
+    FOR    ${cmp}    IN    @{COMPONENTS}
+        Log To Console    ${cmp} - ${COMPONENTS.${cmp}}
+    END
+    Create DataScienceCluster CustomResource Using Test Variables
+    ${yml} =    Get File    ${file_path}dsc_apply.yml
+    Log To Console    Applying DSC yaml
+    Log To Console    ${yml}
+    Run    oc apply -f ${file_path}dsc_apply.yml
+    Remove File    ${file_path}dsc_apply.yml
+    FOR    ${cmp}    IN    @{COMPONENTS}
+        IF    "${COMPONENTS.${cmp}}" == "True"
+            Component Should Be Enabled    ${cmp}
+        ELSE IF    "${COMPONENTS.${cmp}}" == "False"
+            Component Should Not Be Enabled    ${cmp}
+        ELSE
+            Fail    msg=Invalid parameters in test-variables.yml
+        END
+    END
+
+Create DataScienceCluster CustomResource Using Test Variables
+    [Documentation]
+    [Arguments]    ${dsc_name}=default
+    ${file_path} =    Set Variable    ods_ci/tasks/Resources/Files/
```

Comment:
@lugi0 you need to give path from task not from ods_ci since jenkins executed task from ods_ci folder

Reply:
fixed